server/repository/src/migrations/v2_05_00/diagnosis_add_to_invoice.rs

```diff
@@ -0,0 +1,28 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "diagnosis_add_to_invoice"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+            ALTER TABLE invoice ADD COLUMN diagnosis_id TEXT REFERENCES diagnosis(id);
+        "#
+        )?;
+
+        // Retranslate all invoices on the next sync, to make sure we capture the diagnosis_id if populated.
```

Comment:
This might be overkill and could be slow on big sites. Wasn't sure if I should try to optimise by querying for Prescriptions, or do we jsut not care?
@andreievg  thoughts?

Reply:
Great point, I think it's fine not to re-integrate (would only be a problem when migrating mSup to omSup before this change, which i think is super unlikely)